app/web/features/communities/events/LongEventCard.tsx

```diff
@@ -128,9 +128,9 @@ const LongEventCard = ({
           image={event.photoUrl || eventImagePlaceholderUrl}
           title={event.title}
           sx={{
-            height: "100%",
-            width: "25%",
-            objectFit: "fill",
+            objectFit: event.photoUrl ? "none" : "contain",
```

Comment:
```suggestion
            objectFit: event.photoUrl ? "cover" : "contain",
```

Otherwise it looks like this:
<img width="1345" height="234" alt="Screenshot 2025-08-03 at 9 49 57 AM" src="https://github.com/user-attachments/assets/61ae4857-52c9-45b9-aa8b-0b7e96c63bbe" />

Reply:
Oops, yeah that makes much more sense